packages/localizations/src/el-GR.ts

```diff
@@ -0,0 +1,692 @@
+      successMessage: 'Η διπλή επαλήθευση με κωδικούς SMS έχει αφαιρεθεί για τον αριθμό τηλεφώνου {{mfaPhoneCode}}',
+    },
+  },
+  backupCodePage: {
+    title: 'Προσθήκη επαλήθευσης με εφεδρικούς κωδικούς',
+    title__codelist: 'Εφεδρικοί κωδικοί',
+    subtitle__codelist: 'Φυλάξτε τους με ασφάλεια και κρατήστε τους μυστικούς.',
+    infoText1: 'Οι εφεδρικοί κωδικοί θα είναι ενεργοποιημένοι για αυτόν τον λογαριασμό.',
+    infoText2:
+      'Φυλάξτε τους εφεδρικούς κωδικούς μυστικούς και αποθηκεύστε τους με ασφάλεια. Μπορείτε να δημιουργήσετε νέους εφεδρικούς κωδικούς εάν υποψιάζεστε ότι έχουν διαρρεύσει.',
+    successSubtitle:
+      'Μπορείτε να χρησιμοποιήσετε έναν από αυτούς για να συνδεθείτε στον λογαριασμό σας, εάν χάσετε την πρόσβαση στη συσκευή επαλήθευσής σας.',
+    successMessage:
+      'Οι εφεδρικοί κωδικοί είναι πλέον ενεργοποιημένοι. Μπορείτε να χρησιμοποιήσετε έναν από αυτούς για να συνδεθείτε στον λογαριασμό σας, εάν χάσετε την πρόσβαση στη συσκευή επαλήθευσής σας. Κάθε κωδικός μπορεί να χρησιμοποιηθεί μόνο μία φορά.',
+    actionLabel__copy: 'Αντιγραφή όλων',
+    actionLabel__copied: 'Αντιγράφηκαν!',
+    actionLabel__download: 'Λήψη .txt',
+    actionLabel__print: 'Εκτύπωση',
+  },
+  deletePage: {
+    title: 'Διαγραφή λογαριασμού',
+    messageLine1: 'Είστε βέβαιος ότι θέλετε να διαγράψετε τον λογαριασμό σας;',
+    messageLine2: 'Αυτή η ενέργεια είναι μόνιμη και μη αναστρέψιμη.',
+    actionDescription: 'Πληκτρολογήστε "Διαγραφή λογαριασμού" παρακάτω για να συνεχίσετε.',
+    confirm: 'Διαγραφή λογαριασμού',
+  },
+  userButton: {
+    action__manageAccount: 'Διαχείριση λογαριασμού',
+    action__signOut: 'Αποσύνδεση',
+    action__signOutAll: 'Αποσύνδεση από όλους τους λογαριασμούς',
+    action__addAccount: 'Προσθήκη λογαριασμού',
+  },
+  organizationSwitcher: {
+    personalWorkspace: 'Προσωπικός Χώρος Εργασίας',
+    notSelected: 'Δεν έχει επιλεγεί οργανισμός',
+    action__createOrganization: 'Δημιουργία Οργανισμού',
+    action__manageOrganization: 'Διαχείριση Οργανισμού',
+  },
+  impersonationFab: {
+    title: 'Είστε συνδεδεμένος ως {{identifier}}',
+    action__signOut: 'Αποσύνδεση',
+  },
+  organizationProfile: {
+    start: {
+      headerTitle__members: 'Μέλη',
+      headerTitle__settings: 'Ρυθμίσεις',
+      headerSubtitle__members: 'Προβολή και διαχείριση μελών του οργανισμού',
+      headerSubtitle__settings: 'Διαχείριση ρυθμίσεων οργανισμού',
+    },
+    profilePage: {
+      title: 'Προφίλ Οργανισμού',
+      subtitle: 'Διαχείριση προφίλ οργανισμού',
+      successMessage: 'Η οργάνωση έχει ενημερωθεί.',
+      dangerSection: {
+        title: 'Κίνδυνος',
+        leaveOrganization: {
+          title: 'Αποχώρηση από τον οργανισμό',
+          messageLine1:
+            'Είστε σίγουρος ότι θέλετε να αποχωρήσετε από αυτόν τον οργανισμό; Θα χάσετε την πρόσβαση σε αυτόν τον οργανισμό και τις εφαρμογές του.',
+          messageLine2: 'Αυτή η ενέργεια είναι μόνιμη και ανεπανόρθωτη.',
```

Comment:
@EmmanouelaPothitou I think this sounds better. "Αυτή η ενέργεια είναι μόνιμη και μη αναστρέψιμη"